rust/frontend/src/server.rs

```diff
@@ -809,7 +810,7 @@ async fn count_collections(
 #[derive(Deserialize, Serialize, ToSchema, Debug, Clone)]
 pub struct CreateCollectionPayload {
     pub name: String,
-    pub configuration: Option<CollectionConfiguration>,
+    pub configuration_json_str: Option<String>,
```

Comment:
why isn't this 
```suggestion
    pub configuration_json_str: Option<CollectionConfiguration>,
```
?

Reply:
this wont work?

Reply:
why not?